src/mgr.sol

```diff
@@ -127,20 +150,20 @@ contract TinlakeManager is LibNote {
         ilk = ilk_;
         wards[msg.sender] = 1;
 
```

Comment:
```suggestion
emit Rely(msg.sender); 
```

Reply:
I adopted it from the MIP21
https://github.com/livnev/rwa-example/pull/8/files#diff-20efe103664fbcfe47bafc2b8deb1cc4fcb9939f45fe32d91c2b0dee0ddc551aR31

Reply:
see https://github.com/centrifuge/tinlake-maker-lib/pull/18#discussion_r565448088